jax/_src/numpy/fft.py

```diff
@@ -252,17 +252,170 @@ def ifftn(a: ArrayLike, s: Shape | None = None,
+    ...   jnp.fft.rfftn(x, s=[3, 3, 4])
+    Array([[[ 78.   +0.j  , -16.  -26.j  ,  26.   +0.j  ],
+            [ 15.  -36.37j, -16.12 +1.93j,   5.  -12.12j],
+            [ 15.  +36.37j,   8.12-11.93j,   5.  +12.12j]],
+    <BLANKLINE>
+           [[ -7.5 -49.36j, -20.45 +9.43j,  -2.5 -16.45j],
+            [-25.5  -7.79j,  -0.6 +11.96j,  -8.5  -2.6j ],
+            [ 19.5 -12.99j,  -8.33 -6.5j ,   6.5  -4.33j]],
+    <BLANKLINE>
+           [[ -7.5 +49.36j,  12.45 -4.43j,  -2.5 +16.45j],
+            [ 19.5 +12.99j,   0.33 -6.5j ,   6.5  +4.33j],
+            [-25.5  +7.79j,   4.6  +5.04j,  -8.5  +2.6j ]]], dtype=complex64)
+
+    When ``s=[3, 5]`` and ``axes=(0, 1)``, size of the transform along ``axis 0``
+    will be ``3``, along ``axis 1`` will be ``5//2+1 = 3`` and dimension along
+    other axes will be same as that of input.
+
+    >>> with jnp.printoptions(precision=2, suppress=True):
+    ...   jnp.fft.rfftn(x, s=[3, 5], axes=[0, 1])
+    Array([[[ 18.   +0.j  ,  26.   +0.j  ,  34.   +0.j  ],
+            [ 11.09 -9.51j,  16.33-13.31j,  21.56-17.12j],
+            [ -0.09 -5.88j,   0.67 -8.23j,   1.44-10.58j]],
+    <BLANKLINE>
+           [[ -4.5 -12.99j,  -2.5 -16.45j,  -0.5 -19.92j],
+            [ -9.71 -6.3j , -10.05 -9.52j, -10.38-12.74j],
+            [ -4.95 +0.72j,  -5.78 -0.2j ,  -6.61 -1.12j]],
+    <BLANKLINE>
+           [[ -4.5 +12.99j,  -2.5 +16.45j,  -0.5 +19.92j],
+            [  3.47+10.11j,   6.43+11.42j,   9.38+12.74j],
+            [  3.19 +1.63j,   4.4  +1.38j,   5.61 +1.12j]]], dtype=complex64)
+
+    For 1-D input:
+
+    >>> x1 = jnp.array([1, 2, 3, 4])
+    >>> jnp.fft.rfftn(x1)
+    Array([10.+0.j, -2.+2.j, -2.+0.j], dtype=complex64)
+  """
   return _fft_core('rfftn', xla_client.FftType.RFFT, a, s, axes, norm)
 
 
-@implements(np.fft.irfftn)
 def irfftn(a: ArrayLike, s: Shape | None = None,
            axes: Sequence[int] | None = None,
            norm: str | None = None) -> Array:
+  """Compute a real-valued multidimensional inverse discrete Fourier transform.
+
+  JAX implementation of :func:`numpy.fft.irfftn`.
+
+  Args:
+    a: input array.
+    s: optional sequence of integers. Specifies the size of the output in each
+      specified axis. If not specified, the dimension of output along axis
+      ``axes[-1]`` is ``2*(m-1)``, ``m`` is the size of input along axis ``axes[-1]``
+      and the dimension along other axes will be the same as that of input.
+    axes: optional sequence of integers, default=None. Specifies the axes along
+      which the transform is computed. If not specified, transform is computed
+      along the last ``len(s)`` axes or along all the axes, if ``s`` is also not
+      specified.
+    norm: string, default="backward". The normalization mode. "backward", "ortho"
+      and "forward" are supported.
+
+  Returns:
+    A real-valued array containing the multidimensional inverse discrete Fourier
+    transform of ``a``.
```

Comment:
Mention the shape here

Reply:
Updated the size of the resulting array.